src/components/shared/TractionInput.vue

```diff
@@ -0,0 +1,87 @@
+<template>
```

Comment:
This is great. It would be good to see a bit more documentation especially related to what each of the props actually does and how they are used.

Reply:
👍